book/02-git-basics/sections/getting-a-repository.asc

```diff
@@ -7,7 +7,25 @@ The second clones an existing Git repository from another server.
 
 ==== Initializing a Repository in an Existing Directory
 
-If you're starting to track an existing project in Git, you need to go to the project's directory and type
+If you're starting to track an existing project in Git, you need to go to the project's directory
```

Comment:
What if we lead into the examples in a bit friendlier of a way? Something like this:

> […] you need to go to the project's directory. If you've never done this, it looks a little different depending on which system you're running.